### PR TITLE
use correct embed... domain in staging and dev

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

**Comment by elbotho** [comment]
when https://github.com/serlo/serlo.org-cloudflare-worker/pull/128 is in production also, we can think about a hotfix to production also.

File: src/helper/cloudflare-worker-domain.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const cloudflareWorkerDomain =
```
Comment (Entkenntnis) [inline src/helper/cloudflare-worker-domain.ts]: This value is exactly the same as `serloDomain`, is a separate cloudflareWorkerDomain necessary here?

**Review by Entkenntnis — CHANGES_REQUESTED** [review]
Would

File: src/helper/cloudflare-worker-domain.ts (hunk 1 above)
Comment (elbotho) [reply src/helper/cloudflare-worker-domain.ts]: Thanks, that's why I did not commit it directly.
I searched for that but didn't find it…
yeah, serloDomain should be fine :)

**Comment by Entkenntnis** [comment]
As a matter of fact, the current production code already uses `embed.serlo.org`, so it will get the changes as soon as cf worker updates.

**Pull request merged**